### DISCUSSION TITLE
创建人和修改人存储的是def_user中的id值,我应该如何进行映射让他返回的是def_user中的username名字

### DISCUSSION THREAD

**Discussion by IReigns** [zuihou/zuihou-admin-cloud#351] [01.常见问题]
![image](https://github.com/user-attachments/assets/3c340ad3-e5df-49ac-8d9d-3b05b773ac19)

**Comment by zuihou** [comment]
需要自己开发，以下思路仅供参考：

1. 连表查询
2. 新增冗余字段，username，保存时存入用户名
3. 采用@Echo进行回显:  https://tangyh.top/doc/util/lamp-echo-starter.html
